src/com/senlainc/warsaw/tyurin/Test.java

```diff
@@ -0,0 +1,208 @@
+        garagePlace1.getSchedule().put(LocalDateTime.of(2023, 7, 15, 12, 0), true);
+        garagePlace1.getSchedule().put(LocalDateTime.of(2023, 7, 20, 12, 0), true);
+        garagePlace1.getSchedule().put(LocalDateTime.of(2023, 7, 25, 12, 0), true);
+        garagePlace1.getSchedule().put(LocalDateTime.of(2023, 7, 30, 12, 0), true);
+
+        GaragePlace garagePlace2 = new GaragePlace();
+        garagePlace2.setId(2);
+        garagePlace2.setNumber(2);
+        garagePlace2.setSpace(10.0);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 01, 12, 0), true);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 05, 12, 0), true);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 10, 12, 0), true);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 11, 13, 0), true);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 15, 12, 0), true);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 20, 12, 0), true);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 25, 12, 0), true);
+        garagePlace2.getSchedule().put(LocalDateTime.of(2023, 7, 30, 12, 0), true);
+
+        GaragePlace garagePlace3 = new GaragePlace();
+        garagePlace3.setId(3);
+        garagePlace3.setNumber(3);
+        garagePlace3.setSpace(10.0);
+        garagePlace3.getSchedule().put(LocalDateTime.of(2023, 7, 01, 12, 0), true);
+        garagePlace3.getSchedule().put(LocalDateTime.of(2023, 7, 05, 12, 0), true);
+        garagePlace3.getSchedule().put(LocalDateTime.of(2023, 7, 10, 12, 0), true);
+        garagePlace3.getSchedule().put(LocalDateTime.of(2023, 7, 15, 12, 0), true);
+        garagePlace3.getSchedule().put(LocalDateTime.of(2023, 7, 20, 12, 0), true);
+        garagePlace3.getSchedule().put(LocalDateTime.of(2023, 7, 25, 12, 0), true);
+        garagePlace3.getSchedule().put(LocalDateTime.of(2023, 7, 30, 12, 0), true);
+
+        Order order1 = new Order();
+        order1.setId(1);
+        order1.setStartDate(LocalDateTime.of(2023, 7, 05, 12, 0));
+        order1.setCompletionDate(LocalDateTime.of(2023, 7, 05, 13, 0));
+        order1.setOrderStatus(OrderStatus.COMPLETED);
+        order1.addCraftsman(ted);
+        order1.addCraftsman(bob);
+
+        Order order2 = new Order();
+        order2.setId(2);
+        order2.setStartDate(LocalDateTime.of(2023, 7, 30, 12, 0));
+        order2.setCompletionDate(LocalDateTime.of(2023, 7, 30, 13, 0));
+        order2.addCraftsman(ted);
+        order2.addCraftsman(bob);
+        order2.setOrderStatus(OrderStatus.CANCELED);
+
+        Order order3 = new Order();
+        order3.setId(3);
+        order3.setStartDate(LocalDateTime.of(2023, 7, 15, 12, 0));
+        order3.setCompletionDate(LocalDateTime.of(2024, 7, 15, 13, 0));
+        order3.addCraftsman(ted);
+        order3.addCraftsman(bob);
+
+        GaragePlaceDAO garagePlaceDAO = GaragePlaceDAO.getInstance();
```

Comment:
You can not operate with dao on this layer, here you can operate only with Service.
So as i wrote in comment before, create and save entities through service methods